src/client/pages/PageRoutes/hooks/useRoutes.tsx

```diff
@@ -28,53 +29,54 @@ export const useRoutes = () => {
   return useMemo(() => {
     const children = (
       <Route element={<PageLayout />} path="/">
-        <Route index element={<Landing />} />
+        <Route element={<Landing />} index />
 
-        <Route path={Routes.Assessment.path.relative} element={<Assessment />}>
-          <Route path={Routes.Cycle.path.relative} element={<Cycle />}>
-            <Route index element={<CycleHome />} />
+        <Route element={<Assessment />} path={Routes.Assessment.path.relative}>
+          <Route element={<Cycle />} path={Routes.Cycle.path.relative}>
+            <Route element={<CycleHome />} index />
 
             {/* Admin */}
-            <Route path={Routes.Admin.path.relative} element={<Admin />}>
-              <Route index element={<Navigate to={Routes.AdminCountries.path.relative} replace />} />
-              <Route path={Routes.AdminInvitations.path.relative} element={<AdminInvitations />} />
-              <Route path={Routes.AdminCountries.path.relative} element={<AdminCountries />} />
-              <Route path={Routes.AdminUserManagement.path.relative} element={<UserManagement />} />
+            <Route element={<Admin />} path={Routes.Admin.path.relative}>
+              <Route element={<Navigate replace to={Routes.AdminCountries.path.relative} />} index />
+              <Route element={<AdminCountries />} path={Routes.AdminCountries.path.relative} />
+              <Route element={<AdminInvitations />} path={Routes.AdminInvitations.path.relative} />
+              <Route element={<AdminLinks />} path={Routes.AdminLinks.path.relative} />
```

Comment:
This is my change. The other changes were made by the pre-commit hooks (which seems to re-order props)